make_portable.sh

```diff
@@ -0,0 +1,156 @@
+#! /bin/bash
+
+out=~/OpenXRay
+def_copy_path=~
+
+#=================================== Функция справки
+
+helps(){
+    whiptail --title  " Help " --msgbox  "  This script will help you easily build the OpenXRay engine and set it up to run. The script was compiled as a result of numerous requests from users who have the same type of minor errors as a result of their little preparation in order to simplify the process of building the engine.
+The following functions are implemented in the Script:
+1) Updating the source code tree.
+2) Building the OpenXRay engine
+3) Unpacking the installation package with the game. (system must have innoextract installed)
+4) Copying game resources" 14 100
+    main
+}
+
+#=================================== Функция обновления
+
+update_src(){
+    git pull
+    main
+}
+
+#=================================== Функция сборки
+
+build(){
+    if (whiptail --title  " Dependencies " --yes-button  "Build" --no-button  "Cancel" --yesno  "   Before building the OpenXRay engine, you should make sure you have the following packages installed:
+
+gcc cmake make libglvnd libjpeg6-turbo ncurses glew sdl2 openal crypto++ libogg libtheora libvorbis lzo lzop libjpeg-turbo
+
+Attention!!! On some distributions, packages may be prefixed with -dev e.g. libglvnd-dev libjpeg6-turbo-dev " 14 130)  then
+
+
+#   rm -f -R bin
+   mkdir -p bin
+   cd bin
+   cmake .. -DCMAKE_BUILD_TYPE=Release \
+   -DCMAKE_INSTALL_PREFIX=/usr \
+   -DCMAKE_INSTALL_LIBDIR=lib
+   make -j5
```

Comment:
`make -j5` - looks strange for me.
I would suggest to use:
`make -j$(nproc)` or `make -j$(nproc || echo 4)`
(`nproc` - is part of coreutils like `ls` or `mkdir`, so I thinks that not need to check if it exists)

Reply:
Thanks, I'll fix that.